introduction.md

```diff
@@ -0,0 +1,2 @@
+I am a 25 year old Developer that loves learning and teadching code and development related technologies.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.